app/controllers/admin/config/site_settings_controller.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+class Admin::Config::SiteSettingsController < Admin::AdminController
+  ADMIN_CONFIG_AREA_ALLOWLISTED_HIDDEN_SETTINGS = %i[
```

Comment:
Maybe we can introduce "experimental settings" or a similar name?

Reply:
I'm not sure how often we will end up using this in practice...if we end up doing it more often then yes sure we can think of alternative names. For now I wanted to make it super obvious what this was doing when I suggested it to Osama